src/processes/follow-topic-process.js

```diff
@@ -0,0 +1,72 @@
+const StreamChat = require('stream-chat').StreamChat;
+
+const {LogError, UserTopic, CommunityMessageFormat} = require('../databases/models');
+
+const followTopicProcess = async (job, done) => {
+  try {
+    console.info('running job register process ! with id ' + job.id);
+    const serverClient = StreamChat.getInstance(process.env.API_KEY, process.env.SECRET);
+    if (process.env.AUTO_WLCM_MSG === 'true') {
+      let data = job.data;
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Consider destructuring `data` directly from `job` for cleaner code.

```diff
- let data = job.data;
+ const { data } = job;
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->